src/util/KVStore/AMOStore.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#include "AMOStore.hpp"
+
+#include <stdlib.h>
+
+#include <map>
+#include <string>
+#include <iostream>
+
+#include "Action.hpp"
+#include "AMOCommand.hpp"
+#include "AMOResponse.hpp"
+#include "KVStore.hpp"
+
+namespace AMOStore {
+using std::string;
+using std::map;
+using std::cout;
+using std::endl;
+
+bool AMOStore::AlreadyExecuted(const AMOCommand::AMOCommand &command) const {
+  return prev_.find(command) != prev_.end();
+}
+
+AMOStore::AMOStore() {
+  kvStore_ = KVStore::KVStore();
+}
+
+AMOResponse::AMOResponse AMOStore::Execute(
+                const AMOCommand::AMOCommand &command) {
+  string s;
+  if (!AlreadyExecuted(command)) {
+    // cout << "New" << endl;
```

Comment:
Are you saving these debugging statements for a reason? If so, put a comment why. Otherwise remove them.